src/invoices/Tags/TagsBadge.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  IconButton,
+  PaneMenu,
+} from '@folio/stripes/components';
+
+import { configTags } from '../../common/resources';
+
+const TagsBadge = ({ tagsToggle, tagsQuantity, tagsEnabled }) => (
+  <PaneMenu>
```

Comment:
is it possible to display 2 pane menus on the same pane? I mean tags badge with another actions

Reply:
![image](https://user-images.githubusercontent.com/43407139/63574951-9dce8d00-c591-11e9-8915-08a81ff5fe86.png)